libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewConfig.m

```diff
@@ -119,6 +125,37 @@ - (void)setAttemptOfflineLoad:(BOOL)attemptOfflineLoad
 
 #pragma mark - Configuration helpers
 
+- (SFSDKAppConfigValidationResult *)validate {
+    SFSDKAppConfigValidationResult *baseResult = [super validate];
+    if (!baseResult.validationSucceeded) {
+        return baseResult;
+    }
+    
+    if (self.startPage == nil) {
+        return [[SFSDKAppConfigValidationResult alloc] initWithValidationSucceeded:NO message:@"No start page configured."];
+    }
+    
+    // Sanity check local URLs against absolute URL values.
+    if (self.isLocal && self.startPageIsAbsoluteUrl) {
+        return [[SFSDKAppConfigValidationResult alloc] initWithValidationSucceeded:NO message:@"Local start page should not be absolute URL."];
+    }
+    
+    // Start page makeup for remote apps beyond this point is subject to conditional configuration.
+    if (!self.isLocal) {
```

Comment:
Same comment I did for Android.
Would the rules be more obvious if we wrote it this way
```obj-c
if (!self.isLocal && self.shouldAuthenticate && self.startPageIsAbsoluteUrl) {
    return [[SFSDKAppConfigValidationResult alloc] initWithValidationSucceeded:NO message:@"Cannot authenticate using an absolute URL for start page."];
}
if (!self.isLocal && !self.shouldAuthenticate && !self.startPageIsAbsoluteUrl) {
    return [[SFSDKAppConfigValidationResult alloc] initWithValidationSucceeded:NO message:@"Cannot configure relative URL start page with deferred authentication."];
}
```

Reply:
I'm fine with that.